src/main/java/com/dungeoncrawler/Javiarenka/staticResources/SkillResources.java

```diff
@@ -0,0 +1,21 @@
+package com.dungeoncrawler.Javiarenka.staticResources;
+
+import com.dungeoncrawler.Javiarenka.character.Skill;
+import com.dungeoncrawler.Javiarenka.character.SkillTarget;
+
+import java.util.Arrays;
+import java.util.List;
+
+public interface SkillResources {
+    static List<Skill> defaultSkills() {
+        List<Skill> defaultSkills = Arrays.asList(
+                new Skill("Allies Healing", 0, 10, true, false,
+                        SkillTarget.ALL_ALLIES, 1, 20, 0, CharacterStatus.DEFAULT, 1),
+                new Skill("Self Healing", 5, 15, false, true,
+                        SkillTarget.ALLY_AND_ENEMY, 2, 5, 10, CharacterStatus.DEFAULT, 1),
+                new Skill("Hit and burn", 10, 0, true, false,
+                        SkillTarget.ENEMY, 3, 15, 5, CharacterStatus.BURNED, 2));
```

Comment:
Te linijki póki co wywalają builda Mavenowskiego, ale to domyślam się, że dlatego, że wszystko jest jeszcze Work In Progress.
Poniżej screen z CI:
![image](https://user-images.githubusercontent.com/39905011/94115037-778cb800-fe49-11ea-973c-67d8625d4520.png)

Reply:
Zdaje się, że boli go to, że nie ma jeszcze CharacterStatus. Może jakiegoś pulla Ci brakuje?

Reply:
tak dokładnie, nie zrobiłem pulla przed commitem